Cargo.toml

```diff
@@ -24,6 +24,10 @@ members = [
   "crates/dpapi-web",
   "crates/dpapi-ws",
   "crates/dpapi-transport",
+  "crates/dpapi-pdu",
+  "crates/dpapi-fuzzing",
+  "crates/dpapi-pdu",
+  "crates/dpapi-fuzzing",
```

Comment:
Duplicated?

Reply:
Oh, yes. I think I accidentally added it during conflicts resolving